docs/source/mocks.rst

```diff
@@ -94,3 +94,25 @@ Setting the value of a struct
     valueA: 1234,
     valueB: true,
   });
+
+
+Setting the value of a mapping (won't affect other keys)
+#######################################################
+
+.. code-block:: typescript
+
+  await myMock.setVariable('myMapping', {
+    myKey: 1234
+  });
+
+
+Setting the value of a nested mapping
+#####################################
+
+.. code-block:: typescript
+
+  await myMock.setVariable('myMapping', {
+    myChildMapping: {
+      myKey: 1234
+    }
```

Comment:
isn't it like this?
```suggestion
  await myMock.setVariable('myMapping', {
    [myChildMapping]: {
      myKey: 1234
    }
```

Reply:
it depends if `myChildMapping` is a variable of just the name of the mapping, in this case, it is the name of the mapping